src/main/surfrdMod.F90

```diff
@@ -416,7 +416,9 @@ subroutine surfrd_special(begg, endg, ncid, ns)
 
     call ncd_io(ncid=ncid, varname='PCT_OCEAN', flag='read', data=pctocn, &
          dim1name=grlnd, readvar=readvar)
-    if (.not. readvar) call endrun( msg=' ERROR: PCT_OCEAN NOT on surfdata file'//errMsg(sourcefile, __LINE__))
+    if (.not. readvar) call endrun( msg= &
+       ' ERROR: PCT_OCEAN NOT on surfdata file but required when running ctsm5.2 or newer; ' // &
+       ' you are advised to generate a new surfdata file using the mksurfdata_esmf tool ' // errMsg(sourcefile, __LINE__))
```

Comment:
Unrelated to this PR, this edit fixes #2218, which also needs to come in sooner rather than later.

Reply:
Open new issue about fsurdat versioning and put in IF TIME ALLOWS and add "next"